libr/anal/fcn.c

```diff
@@ -839,8 +839,12 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			{
 				bool must_eob = anal->opt.eobjmp;
 				if (!must_eob) {
-					RIOSection *s = anal->iob.section_vget (anal->iob.io, addr);
-					RIOSection *d = anal->iob.section_vget (anal->iob.io, op.jump);
+					SdbList *secs = anal->iob.section_vget_secs_at (anal->iob.io, addr);
+					SdbList *secsj = anal->iob.section_vget_secs_at (anal->iob.io, op.jump);
+					RIOSection *s = secs ? ls_pop (secs) : NULL;
```

Comment:
isn't ls_pop returning NULl already if secs is NULL? this ternary operator seems unnecessary to me